### ISSUE TITLE
Implementation Units

### ISSUE THREAD

**Issue by DavidCraftDev** [FixlTV/discord-for-school#3]
Ich würde eine Implementation von Units gut finden womit der Stundenplan Automatisch geladen wird und auch Vertretungen und Ausfälle angezeigt werden

**Comment by FixlTV** [comment]
Joa, könnt ich schon mal machen, nur meine Schule hat das ned und da wirds dann bissl schwer

**Comment by DavidCraftDev** [comment]
Könnte dir Temporär meine Anmeldedaten bereitstellen

**Comment by FixlTV** [comment]
Ja, kannst gern mal machen, ich schau dann ob und wenn ja wie es sich umsetzen lässt. Kann aber halt ne Zeit dauern, weils bei uns grad mit Tests ganz nett abgeht
